src/main/java/com/github/argon/sos/moreoptions/game/ui/GridRow.java

```diff
@@ -34,6 +34,24 @@ public void initGrid(List<Integer> columnWidths) {
         }
     }
 
+    public void initGrid(List<Integer> columnWidths, Integer width) {
```

Comment:
I guess this can be somewhat combined with the other initGrid method and overloading =D

Reply:
true, gonna look into it